fv3gfs-physics/Makefile

```diff
@@ -0,0 +1,72 @@
+include ../docker/Makefile.image_names
+
+DOCKER_BUILDKIT=1
```

Comment:
Not about this PR, but I'm hoping I can refactor the tests so that this logic (duplicated in each subproject with savepoint tests) can be greatly reduced.